files/.prettierignore

```diff
@@ -23,3 +16,4 @@
 /package.json.ember-try
 /package-lock.json.ember-try
 /yarn.lock.ember-try
+/pnpm-lock.yaml.ember-try
```

Comment:
ember-try uses [`pnpm-lock.ember-try.yaml`](https://github.com/ember-cli/ember-try/blob/master/lib/dependency-manager-adapters/pnpm.js#L15C31-L15C55) FYI.

I think the only ember-try related entries that are needed are:

```
/.node_modules.ember-try/
/pnpm-lock.ember-try.yaml
```

Because _files_ with an `.ember-try` extension would never be picked up by Prettier in the first place.

Reply:
ah ok! so we should move to just `*.ember-try*`, yeah? that'd cover all of these?

Reply:
I made https://github.com/embroider-build/addon-blueprint/pull/168 as follow-up PR, hope that's okay!